README.md

```diff
@@ -69,6 +69,40 @@ Connector.getDAL(datasetId).
   });
 ```
 
+#### listen
+
+The Data Access Layer object for a collection has a `listen` function that ensures that the CRUDL topics for the data set are subscribed.
+
+```javascript
+
+/**
+* This function has access to the mongoose Model associated with the data set
+* through the "model" property
+*
+* @returns {Promise}
+*/
+function customListFunction() {
+  return this.model.list();
+}
+
+
+var datasetId = "workorders";
+
+Connector.getDAL(datasetId).
+  then(function(_dal) {
+    
+    //This function will subscribe to the CRUDL topics for "workorders"
+    //The "list" topic will execute the "customListFunction"
+    _dal.listen(":data", mediator, {
+        list: customListFunction
```

Comment:
[Question] Why someone would like to have custom list function? I'm missing context here :-(